src/scenes/CollectionGalleryPage/CollectionGalleryPage.tsx

```diff
@@ -32,6 +34,28 @@ function CollectionGalleryPage({ collectionId, username, queryRef }: CollectionG
     });
   }, [username, collectionId, track]);
 
+  const ePress = useKeyDown('e');
+
+  // FIXME: Do we also want the escape/backspace keypress to trigger navigation to /{username}? Not in spec
```

Comment:
Flagging: This is not in spec but it feels in line with existing navigation functionality. So I've added it below